Source/FieldSolver/FiniteDifferenceSolver/FiniteDifferenceSolver.H

```diff
@@ -53,6 +53,15 @@ class FiniteDifferenceSolver
         void ComputeDivE ( const std::array<std::unique_ptr<amrex::MultiFab>,3>& Efield,
                            amrex::MultiFab& divE );
 
+        void EvolveBPML ( std::array< amrex::MultiFab*, 3 > Bfield,
+                      std::array< amrex::MultiFab*, 3 > const Efield,
+                      amrex::Real const dt );
```

Comment:
I think you can likely make these member functions `const` (since they change no value stored in the class) and potentially inline.
```suggestion
        void EvolveBPML ( std::array< amrex::MultiFab*, 3 > const Bfield,
                          std::array< amrex::MultiFab const*, 3 > const Efield,
                          amrex::Real const dt ) const;
```